xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/service/dto/DomainEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.icthh.xm.commons.domain.event.service.dto;
+
+import com.icthh.xm.commons.domain.event.domain.ValidFor;
+import lombok.Data;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+public class DomainEvent {
```

Comment:
let's override method toString and decide what is important to dispaly

Reply:
added all required fields to toString() method